packages/flutter/test/widgets/stack_test.dart

```diff
@@ -453,6 +453,7 @@ void main() {
         textDirection: TextDirection.ltr,
         child: Center(
           child: Stack(
+            overflow: Overflow.visible,
```

Comment:
Why this?

Reply:
I was trying to revert the change in https://github.com/flutter/flutter/pull/61366. I can certainly remove this line but this line could test that overflow won't cause any compile error yet. Would you like me to remove this line?